lib/internal/make-api-call.js

```diff
@@ -93,6 +98,7 @@ exports.inject = function(options) {
       until: function(response) {
         return !(
             response == null
+            || response.status === 403  // Geolocation API only.
```

Comment:
Rate limiting for the geolocation API is handled with a 403 status. I've updated this check to only treat 403s as a retry when it's the Geolocation API.